packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -97,25 +97,22 @@ function addAnnotation(element: HTMLDivElement, annotation: Annotation): void {
  * @param annotationUID - The unique identifier for the annotation.
  */
 function removeAnnotation(
-  element: HTMLDivElement,
-  annotationUID: string
+  annotationUID: string,
+  element?: HTMLDivElement
 ): void {
-  const annotationManager = getViewportSpecificAnnotationManager(element);
+  let annotationManager = getDefaultAnnotationManager();
+  if (element) {
```

Comment:
I think this is a great solution that makes sense, allows us to have the annotation-only-api whilst also allowing for element specific managers if we ever need them. I know @swederik  is away, but want to tag him for visibility about this decision (sure we can merge before he sees it, can always make another change).